datafusion/core/tests/sqllogictests/test_files/information_schema.slt

```diff
@@ -152,3 +155,213 @@ query R
 SHOW TIMEZONE
 ----
 datafusion.execution.time_zone +00:00
+
+
+# information_schema_describe_table
```

Comment:
I also like the slt tests as it is easier to see what is covered and what is not in this file